spark

```diff
@@ -12,6 +12,11 @@
  * this class mainly acts as a passthru to the framework itself.
  */
 
+/**
+ * @var bool
```

Comment:
Is this a valid docblock for a `define()` statement?

Reply:
Yes. 
https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/var.html
https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#518-var

Reply:
Neither of those links provides any information on using `@var` on a define that I can see. Possibly having trouble locating it on mobile, but could you paste the relevant quote here?

Reply:
> You may use the @var tag to document the "Type" of the following "Structural Elements":
>    Constants, both class and global scope